pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/External/AdminerController.php

```diff
@@ -86,12 +87,10 @@ public function proxyAction(Request $request)
                         $content .= file_get_contents($this->adminerHome . 'designs/konya/adminer.css');
                         $content .= file_get_contents(PIMCORE_WEB_ROOT . '/pimcore/static6/css/adminer-modifications.css');
                     }
-
-                    $response->setContent($content);
                 }
             }
 
-            return $response;
+            return new Response($content, 200, array_merge($headers, $this->getAdminerSetHeaders()));
```

Comment:
Wouldn't `$response->headers->set($header, $value)` do the same as the `replace` argument defaults to true? Would be nicer than falling back to the array syntax IMHO. Maybe we could add an additional method which applies the headers from `getAdminerSetHeaders()` to a response object and use that method from both actions.